src/routes/trainerMatchingRoute.js

```diff
@@ -0,0 +1,25 @@
+const express = require("express");
+
+const { trainerMatchingController } = require("../controllers");
+const { asyncWrap } = require("../middlewares/errorHandler");
+
+const trainerMatchingRouter = express.Router();
+
+trainerMatchingRouter.get(
+  "/",
+  asyncWrap(trainerMatchingController.getTrainerProduct)
+);
+trainerMatchingRouter.post(
+  "/",
+  asyncWrap(trainerMatchingController.postTrainerProduct)
+);
+trainerMatchingRouter.delete(
+  "/",
+  asyncWrap(trainerMatchingController.deleteTrainerProduct)
+);
+trainerMatchingRouter.get(
+  "/detail",
+  asyncWrap(trainerMatchingController.getTrainerProductDetail)
```

Comment:
```suggestion
trainerMatchingRouter.get(
  "/:trainerProductId,
  asyncWrap(trainerMatchingController.getTrainerProductDetail)
```

상세 정보를 조회할 것이라면, productId를 프론트로부터 받아오는 것이 일반적입니다. 로직상 불가능한 상황이라면 알려주세요.

Reply:
위의 get 요청에서 쿼리로 sort 형식을 받아오기로 해서요 혹시 /detail/:trainerProductId이렇게 받아도 될까요?